tests/test_metrics.py

```diff
@@ -1,9 +1,17 @@
 from typing import Callable
 import pytest
 import numpy as np
+import pandas as pd
 
 import modelskill.metrics as mtr
 
+@pytest.fixture
```

Comment:
Since obs_series and mod_series are only used a single place I think it is better to define them inside test_pr

Reply:
used on a single place _so far_ ... if this makes running the tests slower I understand, but if not, will it no be easier for future tests to already have it as a pytest fixture? I can move it if you really think is better though.